app/src/cc/arduino/view/preferences/Preferences.java

```diff
@@ -833,6 +838,8 @@ private void savePreferencesData() {
 
     PreferencesData.setBoolean("ide.accessible", accessibleIDEBox.isSelected());
 
+    PreferencesData.setBoolean("build.load_sketch_libraries",loadSketchLibrariesBox.isSelected());
```

Comment:
Should this be `use_sketch_libraries` instead? These libraries are not loaded by default, they are made available and used when required. "Use" is also the term used in the checkbox text?